app/models/deck.rb

```diff
@@ -0,0 +1,37 @@
+class Deck < ActiveRecord::Base
+  has_many :cards
+  belongs_to :user
+
+  before_validation :set_default_description, on: [:create]
+  validates :user_id, presence: true
+
+  def self.of_user(id)
+    where(user_id: id)
+  end
+
+  def self.make_main(id)
+    @deck = Deck.find(id)
+    if find_main(id) != nil
+      old_main = find_main(id)
+      old_main.update_attributes(main: false)
+    end
+    @deck.update_attributes(main: true)
+  end
+
+  def self.make_common(id)
+    deck = Deck.find(id)
+    deck.update_attributes(main: false)
+  end
+
+  private
+
+  def set_default_description
+    if self.description.blank?
+      self.description = "Новая колода"
+    end
+  end
+
+  def self.find_main(id)
```

Comment:
Unused method argument - `id`. If it's necessary, use `_` or `_id` as an argument name to indicate that it won't be used. You can also write as `find_main(*)` if you want the method to accept any arguments but don't care about them.